cmd/up/up.go

```diff
@@ -361,20 +364,23 @@ func Up() *cobra.Command {
 }
 
 // AddArgs sets the args as options and return err if it's not compatible
-func (o *UpOptions) AddArgs(cmd *cobra.Command, args []string) error {
+func (o *UpOptions) AddArgs(args []string, oktetoManifest *model.Manifest) error {
 
-	maxV1Args := 1
-	docsURL := "https://okteto.com/docs/reference/cli/#up"
-	if len(args) > maxV1Args {
-		cmd.Help()
-		return oktetoErrors.UserError{
-			E:    fmt.Errorf("%q accepts at most %d arg(s), but received %d", cmd.CommandPath(), maxV1Args, len(args)),
-			Hint: fmt.Sprintf("Visit %s for more information.", docsURL),
+	if len(args) == 1 {
+		if _, ok := oktetoManifest.Dev[args[0]]; ok {
```

Comment:
@jLopezbarb Let's say we pass in `okteto up ["echo", "hello"]`. Would there be any sort of weird error or would the `ok` just be `false`

Reply:
There shouldn't be any errors here. Also, you can do the full test by building the binary with `make build`

Reply:
I'm able to build the binary but I can't really run commands since I don't have access to the okteto cloud.

Reply:
@jLopezbarb Let's create an account in Okteto Cloud for @abs007

Reply:
> @jLopezbarb Let's say we pass in okteto up ["echo", "hello"] through the cli. Would there be any sort of weird error or would the ok just be false
 
Let's say we are running the command you proposed. Since there are no devs with the name `echo` we will set `echo hello` as the command for the dev that the user will select.

> @jLopezbarb Let's create an account in Okteto Cloud for @abs007

I'll work on it. @abs007 I will contact you in case I need anything and will notify  you whenever is ready

Reply:
>Let's say we are running the command you proposed. Since there are no devs with the name echo we will set echo hello as the command for the dev that the user will select.

Ah, okay. I was just wondering if there might be any issues if the user supplies the commands in an array form.

Reply:
@abs007 instead of checking for a `dev container` name, let's use a flag `--command` to overwrite the command. I find the current approach a little ambiguous. And it might get worse if we support activating several dev containers with a single up command like this `okteto up api frontend`

Reply:
I'm guessing this also means reverting `AddArgs()` to it's original functionality?

Reply:
Should I reopen #3211 then?

Reply:
Let's reopen it until we finish implementing this feat and will take a look when we close it

Reply:
Sounds good 👍

Reply:
@jLopezbarb  Should I reopen #3211 now?